lib/fastlane/plugin/firebase_app_distribution/client/firebase_app_distribution_api_client.rb

```diff
@@ -198,33 +205,45 @@ def get_udids(app_id)
 
       private
 
-      def v1_apps_url(app_id)
+      def v1alpha_apps_url(app_id)
         "/v1alpha/apps/#{app_id}"
       end
 
-      def release_notes_create_url(app_id, release_id)
-        "#{v1_apps_url(app_id)}/releases/#{release_id}/notes"
+      def v1_apps_url(app_name)
+        "/v1/#{app_name}"
       end
 
-      def enable_access_url(app_id, release_id)
-        "#{v1_apps_url(app_id)}/releases/#{release_id}/enable_access"
+      def aab_info_url(app_name)
+        "#{v1_apps_url(app_name)}/aabInfo"
       end
 
-      def binary_upload_url(app_id)
-        "/app-binary-uploads?app_id=#{app_id}"
+      def update_release_notes_url(release_name)
+        "/v1/#{release_name}"
       end
 
-      def upload_status_url(app_id, app_token)
-        "#{v1_apps_url(app_id)}/upload_status/#{app_token}"
+      def distribute_url(release_name)
+        "/v1/#{release_name}:distribute"
+      end
+
+      def binary_upload_url(app_name)
+        "/upload#{v1_apps_url(app_name)}/releases:upload"
+      end
+
+      def upload_status_url(operation_name)
+        "/v1/#{operation_name}"
       end
 
       def get_udids_url(app_id)
-        "#{v1_apps_url(app_id)}/testers:getTesterUdids"
+        "#{v1alpha_apps_url(app_id)}/testers:getTesterUdids"
+      end
+
+      def app_name(app_name)
+        "projects/#{project_number(app_name)}/apps/#{app_id}"
       end
 
-      def get_upload_token(project_number, app_id, binary_path)
+      def get_operation_name(app_name, binary_path)
         binary_hash = Digest::SHA256.hexdigest(read_binary(binary_path))
-        CGI.escape("projects/#{project_number}/apps/#{app_id}/releases/-/binaries/#{binary_hash}")
+        CGI.escape("projects/#{project_number(app_name)}/apps/#{app_id}/releases/-/operations/#{binary_hash}")
```

Comment:
I don't think there should be any escaping - and it's just 
`"#{app_name}/releases/-/operations/#{binary_hash}"`

Reply:
Yeah agreed, not sure why it was there to behind with.